### DISCUSSION TITLE
동시에 여러링크를 클릭하면 오류가 나는게 정상인가요?

### DISCUSSION THREAD

**Discussion by jeongsoohwan92** [changdoc/naver-vaccine-macro#127] [Q&A]
![image](https://user-images.githubusercontent.com/60769367/127280469-710c9654-a9e1-4ede-8307-f819bfbc8b1a.png)

모든 링크 열기 한 다음 하나를 자동 예약하다가 메인홈의 설명대로 다른 링크에서 페이지 새로고침 눌러서 

5개를 동시에 돌렸는데요 벤 먹었습니다 ㅎㅎ 어쩔수없이 한번에 하나의 병원만 가능한건가요??

**Comment by changdoc** [comment]
네 여러 페이지에서 동시에 신청 페이지에 접근하면 계정이 차단 당하게 되어 있습니다.
관련 내용은 여기서 확인 하시면 됩니다.
https://github.com/changdoc/naver-vaccine-macro/issues/42

**Reply by jeongsoohwan92** [reply]
감사합니다 내용 확인했습니다~~
